src/elements/content-sidebar/activity-feed/common/activity-message/ActivityMessage.scss

```diff
@@ -1,6 +1,7 @@
 @import '../../../../../styles/variables';
 
 .bcs-ActivityMessage {
+    margin-top: 6px;
```

Comment:
Layout like this should usually be handled by the parent component, as this leaf component can be used in several different contexts and experiences.